docs/remote/codespaces.md

```diff
@@ -54,14 +54,10 @@ In addition to the Quickstarts, there are How-to guides, which go into more deta
 * [How-to guide: Visual Studio Code](https://docs.microsoft.com/visualstudio/codespaces/how-to/vscode) - Create and configure a Codespace from the VS Code client.
 * [How-to guide: Browser](https://docs.microsoft.com/visualstudio/codespaces/how-to/browser) - Create and configure a Codespace with only a browser.
 
-## Self-hosted environments
-
-If you already have a working development environment, you can connect your own [self-hosted environments](https://docs.microsoft.com/visualstudio/codespaces/how-to/self-hosting-vscode) to Visual Studio Codespaces. This lets you and your team have the same consistent development experience, whether you are working on a managed cloud-hosted environment or your own infrastructure.
-
 ## Extension authors
 
-The VS Code extension API hides most of the implementation details of running remotely so many extensions will just work in Visual Studio Codespaces environments without any modification. However, we recommend that you test your extension in a Codespace to be sure that all of its functionality works as expected. See the article on [Supporting Remote Development and Visual Studio Codespaces](/api/advanced-topics/remote-extensions.md) for details.
+The VS Code extension API hides most of the implementation details of running remotely so many extensions will just work in GitHub Codespaces environments without any modification. However, we recommend that you test your extension in a Codespace to be sure that all of its functionality works as expected. See the article on [Supporting Remote Development and GitHub Codespaces](/api/advanced-topics/remote-extensions.md) for details.
 
 ## Questions or feedback
 
-If you have questions, you can consult the Visual Studio Codespaces [Troubleshooting guide](https://docs.microsoft.com/visualstudio/codespaces/resources/troubleshooting) and [FAQ](https://docs.microsoft.com/visualstudio/codespaces/resources/faq). If you want to provide [Feedback](https://docs.microsoft.com/visualstudio/codespaces/resources/feedback), you can enter issues in the Visual Studio Codespaces GitHub [repository](https://github.com/MicrosoftDocs/vscodespaces).
+If you have questions, you can consult the [Troubleshooting guide](https://docs.microsoft.com/visualstudio/codespaces/resources/troubleshooting) and [FAQ](https://docs.microsoft.com/visualstudio/codespaces/resources/faq). If you want to provide [Feedback](https://docs.microsoft.com/visualstudio/codespaces/resources/feedback), you can enter issues in the GitHub Codespaces GitHub [repository](https://github.community/c/codespaces-beta/45).
```

Comment:
Should troubleshooting go to https://docs.github.com/en/github/developing-online-with-codespaces/troubleshooting-your-codespace now?

Reply:
I think we can also remove the FAQ and Feedback links.

Reply:
Yeah I think it makes sense to update that troubleshooting link! Updated that link and removed the FAQ and Feedback links as well.